From 186c167dbbf72b04d4c7dca633cd6ea7f97ed653 Mon Sep 17 00:00:00 2001
From: Jonathan Helmus <jjhelmus@gmail.com>
Date: Fri, 9 Feb 2018 14:35:43 -0600
Subject: [PATCH 2/2] add additional cuda rpaths

Add additional rpaths so that the CUDA libraries can be found be the linker.
---
 tensorflow/core/platform/default/build_config/BUILD | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/tensorflow/core/platform/default/build_config/BUILD b/tensorflow/core/platform/default/build_config/BUILD
index f746b15fee..0183dae758 100644
--- a/tensorflow/core/platform/default/build_config/BUILD
+++ b/tensorflow/core/platform/default/build_config/BUILD
@@ -185,6 +185,12 @@ cc_library(
         "//conditions:default": [
             "-Wl,-rpath,../local_config_cuda/cuda/lib64",
             "-Wl,-rpath,../local_config_cuda/cuda/extras/CUPTI/lib64",
+            "-Wl,-rpath,../local_config_cuda/cuda/lib",
+            "-Wl,-rpath,../local_config_cuda/cuda/extras/CUPTI/lib",
+            "-Wl,-rpath,/usr/local/cuda/lib64",
+            "-Wl,-rpath,/usr/local/cuda/lib64/stubs",
+            "-L/usr/local/cuda/lib64",
+            "-L/usr/local/cuda/lib64/stubs",
         ],
     }),
     deps = [
-- 
2.11.1

